ibm_catalog.json

```diff
@@ -0,0 +1,126 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-appid",
+      "label": "IBM AppID",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "reference_architecture",
```

Comment:
`reference_architecture` is not a valid tag. Run `ibmcloud catalog filter options --all` for list of tags. I suggest to add `security` tag